app/src/lib/parser/function_words.js

```diff
@@ -13,10 +15,12 @@ export const function_words = [
 	'of',
 	'probably',
 	'should',
+	'start',
```

Comment:
There is a word 'start' in the ontology, but the way it is written in the phase 1 is different. 'start' is always used in the analyzer to determine the inceptive aspect of a verb and can be treated as a function word.